### DISCUSSION TITLE
Suggestive auto complete when typing computer names.

### DISCUSSION THREAD

**Discussion by brolifen** [lithnet/access-manager#189] [Ideas]
When you start to type a computer name you could perhaps get an auto complete list of similar named computers to choose from. This would also tell you whether or not you even have access to that computer to begin with if nothing would show up. It could perhaps also work with substrings so you could type "VDI" for servers like SRV-APAC-VDI-01 and so on for quick filtering and picking.

**Comment by ryannewington** [comment]
@brolifen 

We've been looking at this idea for a while. We've got a few challenges in 1) making sure it doesn't get used as an enumeration vector and 2) LDAP calls are too slow and expensive for an auto complete function to work well. It also can put a lot of strain on the DCs nearby.

It's not that we won't implement this or its impossible, but yeah, it's a bit of a tricky one.
